tests/test_fileio.py

```diff
@@ -108,3 +109,91 @@ def test_analogs(idx, names, expected_shape, expected_values, extension):
 def test_analogs_to_csv():
     """Assert analogs's to_csv method."""
     compare_csv(filename=ANALOGS_CSV, kind='analogs')
+
+
+def test_slicing_data():
+    c = Analogs3d.from_c3d(MARKERS_ANALOGS_C3D)
+    n_frames = c.shape[2]
+    analog1 = 'Delt_ant.EMG1'
+    analog2 = 'Delt_med.EMG2'
+    analog1_idx = c.get_index(analog1)[0]
+    analog2_idx = c.get_index(analog2)[0]
+
+    # Accessing all values
+    a = copy.copy(c)
```

Comment:
why not `a.copy()`? I think we do no need to import copy

Reply:
Wasn't aware of this method!